lib/Bio/KBase/HandleMngr/HandleMngr.psgi

```diff
@@ -0,0 +1,22 @@
+use Bio::KBase::HandleMngr::HandleMngrImpl;
+
```

Comment:
Do we want to commit the psgi?  I think it may be generated by kb-sdk compile.

Reply:
It is, but I'm trying to avoid running the kb-sdk command